java/code/src/com/suse/manager/model/hub/HubManager.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2024 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
```

Comment:
Red Hat? :)

Reply:
I don't know, maybe we've always been wrong, but this text is what we have in our license file template for CheckStyle and we strictly check for that:

https://github.com/uyuni-project/uyuni/blob/master/java/buildconf/LICENSE.txt

Reply:
That is a good point, I think I have created already some files without the red hat block, I can try to find. 
@juliogonzalez or @mcalmer  do you know anything about this licensing? Should we keep red hat even for new files created now?

Reply:
Keep in mind  I am not a lawyer, but my opinion:

- License: Yes, keep it as is.
- Copyright/Trademark warnings: Not really, provided the code is 100% our creation then probably you can use `SUSE LLC` only.

But I think you should check with the legal department as well, in case I am missing something :-)

Reply:
we did a check with legal already month ago. we also removed the trademark block from checkstyle

Reply:
Good! I was not aware of it, or did not remember. But it makes sense, of course :-)

Reply:
I fixed the headers and updated my IntelliJ copyright template.